sormas-backend/src/main/java/de/symeda/sormas/backend/sample/PathogenTest.java

```diff
@@ -113,6 +119,16 @@ public void setTestedDiseaseDetails(String testedDiseaseDetails) {
 		this.testedDiseaseDetails = testedDiseaseDetails;
 	}
 
+	@ManyToOne(cascade = {})
+	@JoinColumn(nullable = true)
```

Comment:
If I'm not mistaken you can ommit the parameters for both annotations because they're the default